integration-tests/oidc/src/main/resources/application.properties

```diff
@@ -1,23 +1,24 @@
 quarkus.keycloak.devservices.create-realm=false
 quarkus.keycloak.devservices.start-command=start --https-client-auth=required --hostname-strict=false --https-key-store-file=/etc/server-keystore.p12 --https-trust-store-file=/etc/server-truststore.p12 --https-trust-store-password=password --spi-user-profile-declarative-user-profile-config-file=/opt/keycloak/upconfig.json
-quarkus.keycloak.devservices.resource-aliases.keystore=server-keystore.p12
```

Comment:
as you are removing `upconfig.json`, I am not certain if this `spi-user-profile-declarative-user-profile-config-file` still exists and needs to be set judging by this https://www.keycloak.org/server/all-provider-config#_user_profile, but https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/userprofile/DeclarativeUserProfileProviderFactory.java is complex for me to determine. we can keep it if you know it does something.

Reply:
@michalvavrik This resource is shipped with the devservice, I used to keep it here because the devservice was not used in this test earlier. This `upconfig.json` is about avoiding Keycloak insisting on configuring additional properties for users like email, etc

Reply:
alright, thanks for info. strange it is not listed but I probably looked at the wrong page.

Reply:
@michalvavrik That is because it is undocumented, it is a non-official property :-)

Reply:
@michalvavrik You may recall you were dealing with QE tests started suddenly failing with the upgrade to the next KC version and this is the resource which helps dealing with those issues :-)

Reply:
gotcha